tests/ffi/sysctl/test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+
+. ../common/prepare.sh
+
+export QM_HOST_REGISTRY_DIR="/var/qm/lib/containers/registry"
+export QM_REGISTRY_DIR="/var/lib/containers/registry"
+
+disk_cleanup
+prepare_test
+reload_config
+
+exec_cmd "podman run -d --rm --replace -d --name ffi-asil \
+          quay.io/centos-sig-automotive/ffi-tools:latest  > /dev/null"
+
+image_id=$(podman images | grep quay.io/centos-sig-automotive/ffi-tools | awk -F " " '{print $3}')
+if [ -z $image_id ];then
+   echo "NO ffi-tools IMAGE!!!"
+   exit 1
+fi
```

Comment:
Please see this comment
https://github.com/containers/qm/pull/450#discussion_r1620591819

Reply:
I saw they are similar.

Reply:
But it makes more readable. Thanks, done!